Source/CesiumRuntime/Private/Cesium3DTileset.cpp

```diff
@@ -2028,12 +2044,15 @@ void ACesium3DTileset::Tick(float DeltaTime) {
         CreateViewStateFromViewParameters(camera, unrealWorldToCesiumTileset));
   }
 
-  const Cesium3DTilesSelection::ViewUpdateResult& result =
-      this->_captureMovieMode
-          ? this->_pTileset->updateViewOffline(frustums)
-          : this->_pTileset->updateView(frustums, DeltaTime);
+  Cesium3DTilesSelection::ViewUpdateResult result;
+  if (this->_captureMovieMode) {
+    TRACE_CPUPROFILER_EVENT_SCOPE(Cesium::updateViewOffline)
+    result = this->_pTileset->updateViewOffline(frustums);
```

Comment:
This will invoke the assignment operator, which will assign the several std::vectors it contains, which will cause new heap allocations every frame. Better to avoid such unnecessary heap allocations.

Reply:
Good catch. Changed this to a const ptr to avoid the copy. I do prefer a const reference, but I did want to get a scoped event around updateView and updateViewOffline separately. If there's a cleaner way, just let me know.